snap/snapcraft.yaml

```diff
@@ -24,6 +24,17 @@ platforms:
     build-on: [arm64]
     build-for: [arm64]
 
+slots:
+  posix-mq-provider:
+    interface: posix-mq
+    path:
+      - /syncmsg
+      - /testmsg
+    permissions:
+      - read
+      - write
+      - create
```

Comment:
The issue only mentions the `mq_open` syscall. In what scenarios are `write` and `create` needed? Could you provide me some `pmqtest` command examples?

Reply:
It's in the [source code](https://git.kernel.org/pub/scm/utils/rt-tests/rt-tests.git/tree/src/pmqtest/pmqtest.c#n497), these flags.

Reply:
Sorry, I misunderstood the flags used by the `mq_open` function and the implementation of  the `read`, `write`, `create` and `delete` permissions of this interface.